templates/google_one_tap_button.html

```
@@ -1,43 +1,46 @@
-<style>
-/* the google icon sometimes disappears from the button, this css hack fixes that */
-#g_id_signin_mobile div:has(> svg) {
-    min-width: 40px;
-    min-height: 40px;
-}
-</style>
+{% if settings.ENABLE_FIREBASE_AUTH %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)